hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -512,6 +512,7 @@ public synchronized void stop() {
         connectionsMap.clear();
         monitors.clear();
         activeConnections.clear();
+        scheduler.shutdownNow();
```

Comment:
I think `scheduler` should be shutdown on `ConnectionManager.shutdown()`, not on `ConnectionManager.stop()`. Otherwise it should be restarted on `ConnectionManager.start()`.